docs/reference/api/platform/ATCmdParser.md

```diff
@@ -1,11 +1,37 @@
 ## ATCmdParser
 
-[Add description here]
+ATCmdParser is an Mbed OS compatible AT command parser. 
+
+It sends and receives AT commands to and from a communication device implementing the [FileHandle](https://os.mbed.com/docs/v5.6/reference/filehandle.html) interface. It also implements AT command parsing, which validates the data format and separate command and data portion of AT transactions. For example, the UARTSerial communication driver implements the [FileHandle](https://os.mbed.com/docs/v5.6/reference/filehandle.html) interface, and you can use it with ATCmdParser to send and receive AT commands. The actual format of AT commands used depends on the communication device used.
```

Comment:
I learned about 3 classes but have no idea what any of them do or how they interact.

Reply:
Hi Sam,
If you are referring to FileHandle and UARTSerial classes, documentation for them are worked upon as separate tasks by other people. I just added the template for FileHandle.md as it was not there. Once they are in, the class documentation for all classes will be available. I'll push a patch with other review fixes and an example ( https://github.com/ARMmbed/mbed-os-example-atcmdparser ) has been created.